lib/css/CssModulesPlugin.js

```diff
@@ -432,6 +456,7 @@ class CssModulesPlugin {
 				throw e;
 			}
 		}
+		console.log(chunk.id);
```

Comment:
Remove or introduce logger support if we want to have it here.

Reply:
Yeah, it is still under WIP, so I will remove it after debug :smile: